network-primitives/bootstrap_node_provisioner.tf

```diff
@@ -69,6 +69,7 @@ resource "null_resource" "setup-bootstrap-nodes" {
       "sudo /subspace/install_docker.sh",
       "sudo chmod +x /subspace/start_netdata_agent.sh",
       "sudo /subspace/start_netdata_agent.sh ${var.netdata_claim_token} ${var.netdata_claim_rooms} boostrap-node-${count.index}"
+      "sudo iptables -I OUTPUT -d 192.168.0.0/16,172.16.0.0/12,10.0.0.0/8 -j DROP",
```

Comment:
We do need to make this persistent though, right?

Reply:
Ideally yes, but I'm not sure how dynamic docker rules are as docker is also using them. Will check those and add the persistence.

Reply:
nvm, they are just docker specific networks. Iptables will be persisted

Reply:
This doesn't seem to work, got another abuse message. Which doesn't make a lot of sense to me since we both fixed configuration on that machine and added these rules.

Should rules be added to FORWARD as well? Since container's network is private and packets are likely forwarded. I have added forward rule manually for now, though I don't like that I don't understand why this is happening on a fresh node.

Please run `sudo iptables -I FORWARD -d 192.168.0.0/16,172.16.0.0/12,10.0.0.0/8 -j DROP` on all machines.